CONTRIBUTING.md

```diff
@@ -0,0 +1,258 @@
+# Contributing to Bevy
+
+Hey, so you're interested in contributing to Bevy!
+Feel free to pitch in on whatever interests you and we'll be happy to help you contribute.
+Ultimately @cart has final say on which changes are merged, but you're welcome to try and convince him.
+
+Check out our community's [Code of Conduct](https://github.com/bevyengine/bevy/blob/main/CODE_OF_CONDUCT.md) and feel free to say hi on [Discord](https://discord.gg/bevy) if you'd like.
+It's a nice place to chat about priorities, ask quick questions and get to know the other contributors and users in a less formal setting.
+
+Read on if you're looking for:
+
+* the high-level design goals of Bevy
+* conventions and informal practices we follow when developing Bevy
+* general advice on good open source collaboration practices
+* concrete ways you can help us, no matter your background or skill level
+
+We're thrilled to have you along as we build!
+
+## Getting oriented
+
+Bevy, like any from-scratch game engine, is a large project!
+It can be a bit overwhelming to start, so here's the bird's-eye view.
+
+The main [Bevy engine org](https://github.com/bevyengine) has 4 important repos:
+
+1. [`bevy`](https://github.com/bevyengine/bevy) where the engine itself lives, and the bulk of development work occurs.
+2. [`bevy-website`](https://github.com/bevyengine/bevy-website) where the [official website](https://bevyengine.org/), release notes and Bevy book are hosted, created using the Zola static site generator for Rust.
+3. [`awesome-bevy`](https://github.com/bevyengine/awesome-bevy) is a central home for community content: tutorials, tools, templates, showcases and crates! Make a PR if you want to showcase your stuff there!
+4. [`rfcs`](https://github.com/bevyengine/rfcs) a place for informal but detailed discussion and design work for elaborate features and revamps.
+
+The `bevy` repo itself contains many smaller subcrates, each of which can be downloaded on their own, and freely replaced to enable a modular architecture.
+Of particular interest, [`bevy_ecs`](./crates/bevy_ecs) is fully functional as a stand-alone ECS, which can be very valuable if you're looking to integrate it with other game engines or use it for non-game executables.
+
+[`bevy_app`](./crates/bevy_app) and [`bevy_tasks`](./crates/bevy_tasks) are also worth calling out separately: the former serves as a nice framework to handle various data-piping needs, while the latter is our own lightweight custom async library.
+
+## What we're trying to build
+
+Bevy is intended as an **accessible**, **Rust-first**, **commercially-viable**, **free and open source** game engine.
+
+We prioritize:
+
+* rapid experimentation over API stability
+* a consistent vision over a democratic decision making process
+* accessible, lightweight workflows over bureaucratic standardization
```

Comment:
I'm not clear on what "accessible, lightweight workflows" means

Reply:
"flexible processes"